### ISSUE TITLE
Duplicated parameter TYPE "HOME"

### ISSUE THREAD

**Issue by lechner** [artyom-poptsov/guile-ics#1]
Hi,

Working on my contacts from Google, I get:
```
ics/fsm/content-line-context.scm:232:6: In procedure content-line:store-param-value:
Duplicated parameter TYPE "HOME"
```
That's presumably because several of my contacts have more than one home phone number.

Looking at [the RFC for vCards](https://www.rfc-editor.org/rfc/rfc6350#section-5.6) I'm not sure property parameters must be unique.

Kind regards
Felix

**Comment by artyom-poptsov** [comment]
Hello!  Could you please provide a minimal example of input vCard data that leads to the error?

Thanks,
-avp

**Comment by artyom-poptsov** [comment]
Meanwhile I re-read the RFC and it indeed says that for example "TYPE" parameter may contain either a list of values, or the parameter may be specified several times in a content line and that means that consequential values are read as the elements of the value list.

I prepared a patch that should solve the problem:
https://github.com/artyom-poptsov/guile-ics/tree/wip-handle-parameter-duplicates

Could you also please check if this works for you?

Thanks,
-avp

**Comment by lechner** [comment]
Hi Artyom,

Actually, my interpretation was hasty. Google Contacts exports vCards that have multiple `TYPE` attributes:
```
BEGIN:VCARD
VERSION:3.0
FN:Bystander
EMAIL;TYPE=INTERNET;TYPE=HOME:bystander@vcard.broken
END:VCARD
```
That card was exported using the `vCard (for IOS Contacts)` option and then anonymized. Initially, I thought it could be a shortcut for two identical emails, but my real-life contact only has one email address listed as "Home" so that may be a bug inside Google.

Kind regards
Felix

P.S. Github did not allow me to attache the `.vcf` file. Please save with CRLF line breaks.

**Comment by lechner** [comment]
As for your proposed fix, it may be worthwhile to alert users when an existing value in memory is being overwritten.

I would like to implement [this converter](https://github.com/rampitec/google2radicale) in GNU Guile, but maybe some information would have been lost without the issue I experienced.

**Comment by artyom-poptsov** [comment]
Hello,

> As for your proposed fix, it may be worthwhile to alert users when an existing value in memory is being overwritten.

I added a log message for this event; not sure if it should has "warning" or "info" level, but I set it to "warning" for now.

-avp

**Comment by artyom-poptsov** [comment]
Hello!

I released Guile-ICS 0.6.0.  Please check if everything works for you as expected and let me know if this issue is resolved.

Thanks!
-avp
